indico/modules/events/management/forms.py

```diff
@@ -434,7 +435,20 @@ def __init__(self, source_event, target_event, set_defaults=False, **kwargs):
         super().__init__(**kwargs)
         self.selected_items.choices = [(option.name, option.friendly_name) for option in visible_options]
         # We disable all cloners that are not available, handle only cloning to new events,
-        # have conflicts with target_event or any of their dependencies has conflicts with target_event
-        self.selected_items.disabled_choices = [option.name for option in visible_options if not option.is_available
-                                                or conflicts[option.name]
-                                                or any(conflicts[dep] for dep in option.requires)]
+        # have conflicts with target_event or any of their dependencies have conflicts with target_event
+        disabled_choices = []
+        reasons = {}
+        for option in visible_options:
+            if not option.is_available:
+                disabled_choices.append(option.name)
+                reasons[option.name] = _('There is nothing to import')
+            elif conflict := conflicts[option.name]:
+                disabled_choices.append(option.name)
+                # Show only the first conflict
+                reasons[option.name] = conflict[0]
```

Comment:
hm, why not show all of them, e.g. separated with linebreaks or semicolons?

Reply:
Added line breaks when there are multiple conflicts (check the last screenshot).